queries/smithy/highlights.scm

```diff
@@ -0,0 +1,116 @@
+
+; Keywords
+
+[
+  "namespace"
+  "service"
+  "structure"
+  "operation"
+  "list"
+  "map"
+  "union"
+  "resource"
+  "set"
+  "metadata"
+  "apply"
+  "for"
+  "with"
+  "enum"
+  "intEnum"
+] @keyword
+
+(simple_type_name) @type.builtin
+"use" @include
+
+[
+  ":"
+  "."
+] @punctuation.delimiter
+
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+  "#"
+] @punctuation.bracket
+
+(node_keywords) @boolean
+(number) @number
+(quoted_text) @string
+[
+  (comment)
+  (documentation_comment)
+] @comment
+[
+  ("@")
+] @operator
+
+(use_statement
+  (absolute_root_shape_id
+    (namespace) @namespace
+    (identifier) @type
+  ))
+
+(trait
+  (shape_id) @attribute
+)
+
+(node_string_value
+  (shape_id) @type
+)
+
+; (trait_body) @local.scope
```

Comment:
is this meant as a note for a possible future `locals.scm`

Reply:
not sure, I'm not very familiar with the locals.scm myself but I am pretty sure this was not meant to be `@local.scope`, it is not a locally scoped variable.